octoprint_octorelay/__init__.py

```diff
@@ -167,6 +167,9 @@ def on_event(self, event, payload):
             self.print_stopped()
         elif event == Events.PRINT_FAILED:
             self.print_stopped()
+        elif hasattr(Events, "CONNECTIONS_AUTOREFRESHED"): # Requires OctoPrint 1.9+
+            if event == Events.CONNECTIONS_AUTOREFRESHED:
+                self._printer.connect()
```

Comment:
perhaps I should also add a setting for user to disable this behaviour just in case

Reply:
not yet